src/main/java/uk/gov/hmcts/reform/cpo/ApplicationParams.java

```diff
@@ -0,0 +1,19 @@
+package uk.gov.hmcts.reform.cpo;
+
+import lombok.Getter;
+import org.springframework.beans.factory.annotation.Value;
+
+import javax.inject.Named;
+import javax.inject.Singleton;
+
+@Named
+@Singleton
+@Getter
+public class ApplicationParams {
+
+    @Value("${ccd.defaultPageSize}")
+    private String defaultPageSize;
+
+    @Value("${ccd.defaultPageNumber}")
+    private String defaultPageNumber;
```

Comment:
you don't need a default page number. Default page number is always 1 of course

Reply:
I think so , the thing is the spect metoin default values etc .

Reply:
sorry, a bit misleading. I meant you don't need to make the default page **configurable**. Default page number is always 1

Reply:
DONE :)